dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
Thanks for the very specific tips. I'm already doing the last one (maintaining a fork). For my use case, this is the only viable option out of the ones you mentioned.

I understand the decision to not pollute the global too. Just thought there's a chance you'd be OK with such PR so that I would not need to maintain the fork. Oh well, it is what it is :)